Density/app/src/main/java/org/cornelldti/density/density/FacilityPage.kt

```diff
@@ -1,22 +1,20 @@
 package org.cornelldti.density.density
 
+import kotlinx.android.synthetic.main.facility_page.*
```

Comment:
This is so wild 😮  (for future reference: https://kotlinlang.org/docs/tutorials/android-plugin.html)